admin/class-h5p-content-admin.php

```diff
@@ -121,13 +121,31 @@ private function load_content($id) {
    * @return boolean
    */
   private function current_user_can_edit($content) {
+    // If you can't edit content, you neither can edit others contents
+    if (!current_user_can('edit_h5p_contents')) {
+      return FALSE;
+    }
     if (current_user_can('edit_others_h5p_contents')) {
       return TRUE;
     }
     $author_id = (int)(is_array($content) ? $content['user_id'] : $content->user_id);
     return get_current_user_id() === $author_id;
   }
 
+  /**
+   * Permission check. Can the current user view the given content?
+   *
+   * @since 1.15.0
+   * @param array $content
+   * @return boolean
+   */
+  private function current_user_can_view($content) {
```

Comment:
Where is `$content` used?
It looks like you could just have used `current_user_can('view_h5p_contents')` directly instead of having this function?

Reply:
My bad. I borrowed from https://github.com/h5p/h5p-wordpress-plugin/pull/97/files#diff-354ca6777c6035a26f859b9837dee0fcR138 where $content is required, so it will be introduced anyway if the other pull request is going to be accepted, too.